ts/state/ducks/userGroups.ts

```diff
@@ -17,13 +18,28 @@ const userGroupSlice = createSlice({
   reducers: {
     refreshUserGroupsSlice(
       state: UserGroupState,
-      action: PayloadAction<{ groups: Array<UserGroupsGet> }>
+      action: PayloadAction<{ groups: Array<UserGroupsRedux> }>
     ) {
       state.userGroups = {};
       action.payload.groups.forEach(m => {
         state.userGroups[m.pubkeyHex] = m;
       });
 
+      return state;
+    },
+    refreshUserGroupDetails(
+      state: UserGroupState,
+      // if passing null, it means delete that entry from redux
+      {
+        payload,
+      }: PayloadAction<{ group: { pubkey: GroupPubkeyType; details: UserGroupsRedux | null } }>
```

Comment:
Not sure about passing null as a convention. I think it would be better to have a separate method for deletion?

Reply:
tweaked